canonical-web-and-design/charmhub.io
# 1820 add index to library page

## Done
- _Add index navigation to the docstring libraries page._
- _Separate `docstring` and `source-code` in 2 views and routes._

## How to QA
- Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/fran-wordpress/libraries/test
- And see it takes you the docstring page. Click on the `index` navigation at the top and see it works as expected.
- Click on the `Source code` button and see you are taken to http://localhost:8045/fran-wordpress/libraries/test/source-code and the page looks fine

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1820

## Screenshots
[if relevant, include a screenshot]
